### PR TITLE
chore(Terraform): output the outbound IP ranges

### PR DESCRIPTION
**Pull request by afeld**
Will make it easier to copy the values into the Eligibility Server pipeline configuration.

### CODE REVIEW

**Comment by afeld** [comment]
[Preview of what the `output` will be](https://calenterprise.visualstudio.com/CDT.OET.CAL-ITP/_build/results?buildId=38595&view=logs&j=5af648c4-4294-55a7-6072-dd70820d3dba&t=42899ae2-6397-58a2-7641-cec3214b180b&l=43)

File: terraform/output.tf [hunk 1]
```diff
@@ -0,0 +1,4 @@
+output "outbound_ip_ranges" {
+  value       = sort([for ip in azurerm_linux_web_app.main.possible_outbound_ip_address_list : "${ip}/32"])
+  description = "The IP address ranges to use to filter requests to this App Service instance, like for the Eligibility Server"
```
Comment (thekaveman) [inline terraform/output.tf:3]: I think this comment is... maybe a little confusing? We use these IPs to filter requests _to the Eligibility Server_, from this App Service instance.

Comment (afeld) [reply terraform/output.tf]: Whoops, good catch!

**Comment by thekaveman** [comment]
Is this more related to #1171, which we are holding off on per [@angela-tran's suggestion](https://github.com/cal-itp/benefits/issues/1142#issuecomment-1363117266)?

**Comment by afeld** [comment]
It's a convenience (and now docs) to clarify what needs to be done when the outbound IPs change. That includes things like the underlying Plan changes, and thus sets us up for things like https://github.com/cal-itp/benefits/pull/1171.

Similarly, for the Eligibility Server to accept requests from our new dev and test App Service instances, we merged this list with the existing one to allow requests from both App Service instances.

**Comment by angela-tran** [comment]
> Is this more related to #1171, which we are holding off on per [@angela-tran's suggestion](https://github.com/cal-itp/benefits/issues/1142#issuecomment-1363117266)?

I think this is definitely helpful for #1171. It seems ok to get these changes in ahead of time.

**Review by thekaveman — APPROVED** [review]
👍

**Pull request merged**